main.js

```diff
@@ -0,0 +1,52 @@
+/*
+* Object for storing global variables
+* possible: input value strings for jquery selections
+* possible: tweet location holder,  geolocation coordiantes when map click
+*
+* */
+
+
+/*
+* Document Ready function
+* holds button click events to activate correct functions
+* */
+
+
+/*
+* Function: getLocation
```

Comment:
Likewise to the comment below, have Google Maps API calls handled by different methods in an object, in case we need to add others later.